### PR TITLE
Initial chapter for R Proxy, Hosted, Group

### PR DESCRIPTION
**Pull request by DarthHater**

### CODE REVIEW

File: chapter-r.asciidoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
+[[r]]
+== R Repositories
+{inall}
+
+[[r-introduction]]
+=== Introduction
+
+http://[R] is a lanuage used for statistical analysis and many machine learning uses. for front-end web development.
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: Couple of typos:

1. `lanuage`
2. `for front-end web development.`

File: chapter-r.asciidoc [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+[[r-download]]
+=== Configuring R Package Download
+
+Once you have set up your repositories for R packages, and installed R, you can
+create a  file to access repository URLs.
+----
+
+----
+
+IMPORTANT: The .bowerrc file can be located in various locations. For global configuration for a specific
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: We can take out all the `.bowerrc` stuff.

File: chapter-r.asciidoc [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+R is typically installed either as just R itself, or R with R Studio. R can be obtained from CRAN at 
+https://cran.r-project.org/[https://cran.r-project.org/], and R Studio can be obtained from 
+https://www.rstudio.com/products/rstudio/download2/[https://www.rstudio.com/products/rstudio/download2/]. You
+will need to install R before you can install and run R Studio, if you choose to do so.
+
+[[r-download]]
+=== Configuring R Package Download
+
+Once you have set up your repositories for R packages, and installed R, you can
+create a  file to access repository URLs.
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: Unneeded space.

File: chapter-r.asciidoc [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+to upload them to a hosted repository on the repository manager. The consumers can then download it via the 
+repository group as documented in <<r-download>>.
+
+Authentication is managed in the same manner as for proxying with anonymous access disabled as documented in
+<<r-download>>.
+
+With this configuration you can run a command such as
+
+----
+curl -v --user 'admin:admin123' --upload-file example_1.0.0.tar.gz http://localhost:8081/repository/r-hosted/src/contrib/example_1.0.0.tar.gz
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: I don't know what we do elsewhere, maybe change to `user:pass` just so we don't tacitly encourage using admin credentials for this kind of thing?

Also, should we at least have a note about the directory structure for binaries being a bit more complex, and that they should refer to the R documentation somewhere for more information?

File: chapter-r.asciidoc [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+https://www.rstudio.com/products/rstudio/download2/[https://www.rstudio.com/products/rstudio/download2/]. You
+will need to install R before you can install and run R Studio, if you choose to do so.
+
+[[r-download]]
+=== Configuring R Package Download
+
+Once you have set up your repositories for R packages, and installed R, you can create a 
+file to access repository URLs.
+----
+
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: Do we still need this blank part here?

File: chapter-r.asciidoc [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+proxied from an upstream repository and cached in the repository manager.
+
+[[r-publish]]
+=== Publishing R Packages
+
+If you are authoring your own packages and want to distribute them to other users in your organization, you have
+to upload them to a hosted repository on the repository manager. The consumers can then download it via the 
+repository group as documented in <<r-download>>.
+
+Authentication is managed in the same manner as for proxying with anonymous access disabled as documented in
```
Comment (fjmilens3) [inline chapter-r.asciidoc]: I'm not sure this is correct, if we're giving them the curl command.

**Comment by mosabua** [comment]
Merged and subsequently deactivatd the import to clean up branches in this repo

**Pull request merged**